aws_lambda_powertools/logging/formatter.py

```diff
@@ -44,16 +22,15 @@ def __init__(self, **kwargs):
 
         Other kwargs are used to specify log field format strings.
         """
-        self.default_json_formatter = kwargs.pop("json_default", json_formatter)
+        self.default_json_formatter = kwargs.pop("json_default", str)
```

Comment:
add a comment that unserializable values will be cast as str

Reply:
@heitorlessa The original code did mention this in the method docstring. I can put in a line comment as well.

Reply:
a line comment suffices ;)